daffodil-core/src/main/scala/org/apache/daffodil/grammar/primitives/SequenceCombinator.scala

```diff
@@ -45,8 +47,27 @@ abstract class SequenceCombinator(sq: SequenceTermBase, sequenceChildren: Seq[Se
 class OrderedSequence(sq: SequenceTermBase, sequenceChildrenArg: Seq[SequenceChild])
   extends SequenceCombinator(sq, sequenceChildrenArg) {
 
+  private lazy val sepMtaGram = sq.sequenceSeparatorMTA
+  // Note that we actually only every use one of these depending on depending
+  // on various factors. If there is an optional separtor and a suspension is
+  // used to unaprse that separator, then we cannot use the sepMtaUnaprser
+  // because it results in nested suspensions, which isn't allowed. In that
+  // case, the suspension ends up handling both the optional separator and
+  // alignment using sepMtaAlignmentMaybe.
+  private lazy val (sepMtaAlignmentMaybe, sepMtaUnparserMaybe) =
```

Comment:
Reads better as:

```
  // Note that we actually only ever use one of these depending on 
  // various factors. If there is an optional separator and a suspension is
  // used to unparse that separator, then we cannot use the sepMtaUnparser
  // because it results in nested suspensions, which isn't allowed. In that
  // case, the suspension ends up handling both the optional separator and
  // alignment using sepMtaAlignmentMaybe.
```